test/TestUtils/project.json

```diff
@@ -17,20 +17,17 @@
       ]
     }
   },
-
   "dependencies": {
     "Microsoft.NETCore.App": {
       "type": "platform",
-      "version": "1.0.0-rc3-4338"
+      "version": "1.0.0"
     },
```

Comment:
The test projects need to run against the sharedfx in the sdk, so 1.0.0 is right here. The TestAssets will run against the sharedfx produced, and there is xunit scenario coverage.

There are multiple reasons here and we had a lot of back and forth when introducing the tests. This prevents the need to create a patched stage0 with the new sharedfx, leaving us in a chicken/egg problem if we introduce a change the sdk isn't yet compatible with. Furthermore, all tests are runnable via the Visual Studio test runner. If you target this to the latest built version, the Visual Studio test runner won't work because it will be running an older dotnet. 

There's no loss in test coverage as we have explicit TestApps in `TestAssets` that run against latest.